discovery_engine/lib/src/domain/models/configuration.dart

```diff
@@ -29,6 +29,8 @@ class Configuration with _$Configuration {
     required String apiKey,
     required String apiBaseUrl,
     required String assetsUrl,
+    required String newsProviderPath,
```

Comment:
:art: I think this should be grouped together and optional

Reply:
Optional as in we should have a default `newsProviderPath` in the engine, if none is provided? What would grouped look like?

Reply:
i'd assume sth like
```dart
class ProviderPaths {
  final String news,
  final String headlines,
}
```
and then `ProviderPaths? providerPaths` instead of `required String newsProviderPath, required String headlinesProviderPath,` in the config class.

since this would require quite some additional changes on the rust and dart side and the ffi, maybe this could be done in a follow up pr instead?